app.py

```diff
@@ -1,9 +1,18 @@
+import dash_bootstrap_components as dbc
 from dash import Dash, Input, Output, dcc, html
 import dash_bootstrap_components as dbc
 
 app = Dash(__name__, external_stylesheets=[dbc.themes.CYBORG])
 
+navbar = dbc.NavbarSimple(
+    brand="PythonGurl",
+    sticky="top",
+    color="#95A5A6",
```

Comment:
Any reason you are specifying the color manually?  The point of adding a theme is that it will pick keep a complementary color scheme.  Something like `color="primary"` would probably be good.